### PR TITLE
Update Vocalink data sets and PDF docs

### PR DESCRIPTION
**Pull request by gunjam**
Hi,

This pull request updates the Vocalink modulus weight table data and PDF documentation to the latest version which will be valid from 21/8/2017.

Thanks!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+.DS_Store
```
Comment (franciscocardoso) [inline .gitignore]: Can you please revert this?

File: dist/data/valacdos-v440.txt [hunk 2]
```diff
@@ -1,4 +1,7 @@
 010004 016715 MOD11    0    0    0    0    0    0    8    7    6    5    4    3    2    1
```
Comment (franciscocardoso) [inline dist/data/valacdos-v440.txt]: The `dist` files are only added at release time. Please remove `dist/data/valacdos-v440.txt`.

File: dist/index.js [hunk 3]
```diff
@@ -346,7 +346,7 @@ var UkModulusChecking = function () {
   }, {
```
Comment (franciscocardoso) [inline dist/index.js]: The `dist` files are only added at release time. Please remove `dist/index.js`.

**Comment by gunjam** [comment]
Hi,

I've amended the commit to remove the changes from `dist/*` and the `.gitignore` file.

Thanks.

**Pull request merged**